staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation_test.go

```diff
@@ -87,6 +124,9 @@ func TestValidateCustomResourceDefinition(t *testing.T) {
 							Storage: false,
 						},
 					},
+					Conversion: &apiextensions.CustomResourceConversion{
+						Strategy: apiextensions.ConversionStrategyType("nop"),
```

Comment:
make this more obviously invalid, like `"invalid"`, or omit this, since we already have a test case for invalid strategy? I thought this was a typo of `None` at first (applies to all `nop` values)

Reply:
yeah, all of them should be `None`